src/Native/System.Native/pal_runtimeinformation.h

```diff
@@ -14,5 +14,6 @@ enum
 {
     ARCH_X86,
     ARCH_X64,
-    ARCH_ARM
+    ARCH_ARM,
+    ARCH_ARM64
```

Comment:
@Priya91 To compile the native bit for arm64 I need to add a #if defined(_ARM64_) to get a successful C compilation. This is where I needed to add the ARCH_ARM64 new enum entry and updated SystemNative_GetUnixArchitecture accordingly. I'm fine throwing a PNSE in SystemNative_GetUnixArchitecture if that's better for the time being.